### PR TITLE
Pin index

### PR DESCRIPTION
**Pull request by cassiemak**
Individual pin layout in the index page all set 
-still need to put in scraped data in with Angular, will merge with Dale once we got the data from Pinterest
- Might have minor css to fix once we have data to play with

Added the Masonry js files and a different size jpg to play with

### CODE REVIEW

File: app/assets/javascripts/angular/angular-config.js.erb [hunk 1]
```diff
@@ -7,3 +7,9 @@ app.config(['$routeProvider',function($routeProvider){
   })
 }])
 
+app.config(['$routeProvider',function($routeProvider){
+  $routeProvider.when('/pin/', {
+    templateUrl: "<%= asset_path('pin-show.html') %>",
+    controller: 'PinCtrl',
```
Comment (h8rry) [inline app/assets/javascripts/angular/angular-config.js.erb]: This controller hasn't been defined yet

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -16,4 +16,5 @@
 //= require angular-animate
 //= require angular-resource
 //= require angular-route
+//= require masonry/jquery.masonry
```
Comment (h8rry) [inline app/assets/javascripts/application.js]: Didn't give me errors

File: app/assets/stylesheets/application.css.scss [hunk 3]
```diff
@@ -13,6 +13,13 @@
  * 
  *= require_tree .
  *= require_self
+ *= require 'masonry/basic'
+ *= require 'masonry/centered'
+ *= require 'masonry/fluid'
+ *= require 'masonry/gutters'
+ *= require 'masonry/infinitescroll'
+ *= require 'masonry/right-to-left'
+ *= require 'masonry/transitions'
```
Comment (h8rry) [inline app/assets/stylesheets/application.css.scss]: Didn't give me errors

File: app/assets/stylesheets/pin-index.css.scss [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+.pin-outer{
+  background-color: white;
+  border: solid $primary-background-color 2px;
+  max-width: 220px;
+  min-height: 300px;
+  background-color: $primary-background-color; 
+}
+
+div.pin{
```
Comment (h8rry) [inline app/assets/stylesheets/pin-index.css.scss:28]: Why put html tag in front of the class name? But you don't do it consistently as well

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -5,6 +5,7 @@
   <%= stylesheet_link_tag    'application', media: 'all' %> 
   <%= javascript_include_tag 'application' %>
   <%= csrf_meta_tags %>
+  <script type="text/javascript" scr="masonry.pkgd.min.js"></script>
```
Comment (h8rry) [inline app/views/layouts/application.html.erb]: Do we need this when we already have the gem?

**Comment by h8rry** [comment]
Did you try deploying it to Heroku to see if it works?

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (cassiemak) [inline app/views/layouts/application.html.erb]: No. Will remove this.

File: app/assets/stylesheets/pin-index.css.scss (hunk 4 above)
Comment (cassiemak) [inline app/assets/stylesheets/pin-index.css.scss:28]: Sometimes using the specific html tag would over-write whatever inherit css

File: app/assets/stylesheets/application.css.scss (hunk 3 above)
Comment (cassiemak) [inline app/assets/stylesheets/application.css.scss]: GG

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (cassiemak) [inline app/assets/javascripts/application.js]: GG

File: app/assets/javascripts/angular/angular-config.js.erb (hunk 1 above)
Comment (cassiemak) [inline app/assets/javascripts/angular/angular-config.js.erb]: I dont need the controller to see the page. We're not using the controller until we have the scrape data

**Comment by cassiemak** [comment]
Has not deploy to Heroku yet. WIll wait till Dale comes back to do it together.

File: app/assets/javascripts/angular/angular-config.js.erb (hunk 1 above)
Comment (h8rry) [inline app/assets/javascripts/angular/angular-config.js.erb]: Even if there's nothing in the controller, you still need to define it. Or, you can omit the controller key. For example,

``` javascript

app.config(['$routeProvider',function($routeProvider){
  $routeProvider.when('/pin/', {
    templateUrl: "<%= asset_path('pin-show.html') %>"
  })
}])
```

File: app/assets/stylesheets/pin-index.css.scss (hunk 4 above)
Comment (h8rry) [inline app/assets/stylesheets/pin-index.css.scss:28]: But class has a higher priority over html tag

**Comment by h8rry** [comment]
You should deploy your changes now to make sure it works on heroku. Only takes 1 command line and 1 minute of wait to deploy. Right? You always want to make sure that your changes works on production as well.

File: app/assets/javascripts/angular/angular-config.js.erb (hunk 1 above)
Comment (cassiemak) [inline app/assets/javascripts/angular/angular-config.js.erb]: Removing controller line

File: app/assets/stylesheets/pin-index.css.scss (hunk 4 above)
Comment (cassiemak) [inline app/assets/stylesheets/pin-index.css.scss:28]: Added all the html tag now

**Comment by cassiemak** [comment]
I have made all the changes post Harry's comment, please approve the pull request. 
Heroku link - https://wdi-fritter.herokuapp.com/#/

**Pull request merged**